pod/perlsub.pod

```diff
@@ -445,8 +445,77 @@ If there are multiple optional positional parameters and not enough
 arguments are supplied to fill them all, they will be filled from left
 to right.
 
-After positional parameters, additional arguments may be captured in a
-slurpy parameter.  The simplest form of this is just an array variable:
+Since Perl 5.43.3, a signature may also declare named parameters.  Like
+positional parameters, these are lexical variables which will contain values
+passed by the caller when the subroutine is invoked.  Whereas positional
+parameters take their values from the corresponding position in the list
+passed by the caller, named parameters are matched by name-value pairs
+from the caller, where the name matches the variable name of the parameter
+but the order of the pairs is not significant.
+
+Named parameters in signatures are currently considered B<experimental>, and
+their behaviour may change in future releases of Perl.  Uses of named
+parameters in a signature will currently emit a warning in the
+C<experimental::signature_named_parameters> category.
```

Comment:
This could instead be a link to that category ie. within the pod  of the experimental pragma (so that the users could conveniently read more if needed).

Reply:
`experimental` isn't (exactly) a core module, but also its documentation doesn't have separate sections per warning category. Nor does `warnings`.